### PR TITLE
Make migrations users/0006 more independent.

### PR DESCRIPTION
**Pull request by mythmon**
@safwanrahman was having trouble with this: adding new fields to our profile model breaks this migration, since it calls all the bits and bobs to create a new profile as well as a new user. The fields would be missing, and things would break.

r?

### CODE REVIEW

File: kitsune/users/migrations/0006_add_migration_user.py [hunk 1]
```diff
@@ -3,16 +3,17 @@
 from south.db import db
 from south.v2 import DataMigration
 from django.db import models
+from django.contrib.auth.hashers import make_password
 
 class Migration(DataMigration):
 
     def forwards(self, orm):
         """Adds a user to be used for migrations."""
         from django.contrib.auth.models import User
```
Comment (rehandalal) [inline kitsune/users/migrations/0006_add_migration_user.py]: you can probably ditch this line too

**Comment by safwanrahman** [comment]
Thanks @mythmon for this patch. I was having problem with Pull Request #2291 . I hope this patch fix the issue.

File: kitsune/users/migrations/0006_add_migration_user.py (hunk 1 above)
Comment (mythmon) [inline kitsune/users/migrations/0006_add_migration_user.py]: Oh, yes.

**Comment by mythmon** [comment]
Feedbacked.

**Comment by rehandalal** [comment]
lgtm r+

**Comment by mythmon** [comment]
193aea1 Make migrations users/0006 more independent.

**Pull request closed without merging**